config/migrations/001_create_users.rb

```diff
@@ -1,10 +1,10 @@
 migration 'create the users table' do
   database.create_table :users do
     primary_key :id
-    String :login, :unique => true
-    String :name, :null => false
+    String :login
```

Comment:
I cannot understand why `login` should not be unique? It's something like your `username`. another `name` is for display.

Reply:
Only two fields `email` and `password` on register page.If set `login` and `name` value with `email`,i think people will be strange when they setting profile page .If it is not reasonable, I will revert to the previous version

Reply:
So to configure on profile page is mandatory? Otherwise, we will have show email address as user's name on pages